### PR TITLE
Optimize app namesapces and fix some typos in gmc e2e test.

### PR DESCRIPTION
**Pull request by gyohuangxin**
## Description

The summary of the proposed changes as long as the relevant motivation and context.

1. Change the app namespapce from `chatqna-$(date +%Y%m%d%H%M%S)` to `opea-app-$(date +%Y%m%d%H%M%S)` to adapt different workloads instead of just chatqna.
2. Fix some typos in gmc e2e tests.

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

`n/a`

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by zhlsunshine — APPROVED** [review]
LGTM

File: .github/workflows/go-e2e.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Set variables
         run: |
           echo "SYSTEM_NAMESPACE=opea-system-$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
-          echo "APP_NAMESPACE=chatqna-$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
+          echo "APP_NAMESPACE=opea-app-(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
```
Comment (zhlsunshine) [inline .github/workflows/go-e2e.yaml:71]: Hi @gyohuangxin, you are missing a `$` symbol.

Comment (zhlsunshine) [inline .github/workflows/go-e2e.yaml]: Hi @gyohuangxin, you are missing a `$` symbol here.

File: .github/workflows/scripts/e2e/gmc_gaudi_test.sh [hunk 2]
```diff
@@ -87,7 +87,7 @@ function validate_chatqna() {
    sleep 120
 
    kubectl get pods -n $APP_NAMESPACE
-   # send request to chatqnA
+   # send request to chatqna
    export CLIENT_POD=$(kubectl get pod -n $APP_NAMESPACE -l app=client-test -o jsonpath={.items..metadata.name})
```
Comment (zhlsunshine) [inline .github/workflows/scripts/e2e/gmc_gaudi_test.sh:91]: Hi @gyohuangxin, if so, I suggest that add a new namespace named `CHATQNA_NAMESPACE` and it should be `CHATQNA_NAMESPACE="${APP_NAMESPACE}-chatqna"`

Comment (zhlsunshine) [reply .github/workflows/scripts/e2e/gmc_gaudi_test.sh:91]: And the same as in `gmc_xeon_test.sh`

**Review by zhlsunshine — APPROVED** [review]
LGTM

File: .github/workflows/scripts/e2e/gmc_gaudi_test.sh (hunk 2 above)
Comment (gyohuangxin) [reply .github/workflows/scripts/e2e/gmc_gaudi_test.sh]: @zhlsunshine Thanks for your suggestion, and I've add the new namespace "CHATQNA_NAMESPACE="${APP_NAMESPACE}-chatqna". 
Besides, this [line of code](https://github.com/opea-project/GenAIInfra/blob/main/.github/workflows/scripts/e2e/gmc_xeon_test.sh#L292) will replace namespace "default" with `APP_NAMESPACE`, but the `APP_NAMESPACE` won't exist after this change, what namespace should it be changed to?

Comment (zhlsunshine) [reply .github/workflows/scripts/e2e/gmc_gaudi_test.sh]: Hi @gyohuangxin, the GMC controller would handle this and it depends on every example. You also can refer to the comment [here](https://github.com/opea-project/GenAIInfra/pull/142#issuecomment-2199191310)

**Review by daisy-ycguo — APPROVED** [review]
lgtm

**Review by zhlsunshine — COMMENTED** [review]
LGTM

**Pull request merged**